### PR TITLE
feat(new-trace): Using fetched issue data to render drawer issue block.

### PR DESCRIPTION
**Pull request by Abdkhan14**
Looks the same, just uses the fetched issue to render links:
Screenshot 2024-03-13 at 2 39 13 PM

### CODE REVIEW

File: static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issue.tsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      <ErrorDot level={fetchedIssue.level} />
+      <ErrorLevel>{fetchedIssue.level}</ErrorLevel>
+      <ErrorTitle>
+        <Link to={generateIssueEventTarget(props.issue, props.organization)}>
+          {fetchedIssue.title}
+        </Link>
+      </ErrorTitle>
+    </ErrorMessageContent>
+  ) : isError ? (
+    <LoadingError message={t('Failed to download attachment.')} />
```
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issue.tsx:54]: failed to fetch attachment

File: static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx [hunk 2]
```diff
@@ -314,30 +306,10 @@ export function TransactionNodeDetails({
...
-            {tn(
-              '%s issue occurred in this transaction.',
-              '%s issues occurred in this transaction.',
-              node.value.errors.length + node.value.performance_issues.length
-            )}
-          </ErrorMessageTitle>
-        </Alert>
+        <IssueList
+          organization={organization}
+          issues={[...node.value.errors, ...node.value.performance_issues]}
```
Comment (JonasBa) [inline static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx:311]: Can you memo this?

File: static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issue.tsx (hunk 1 above)
Comment (Abdkhan14) [reply static/app/views/performance/newTraceDetails/traceDrawer/details/issues/issue.tsx:54]: Oops

File: static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx (hunk 2 above)
Comment (Abdkhan14) [reply static/app/views/performance/newTraceDetails/traceDrawer/details/transaction.tsx:311]: Yep

**Review by JonasBa — APPROVED** [review]

**Pull request merged**